Documentation/Books/Manual/ReleaseNotes/KnownIssues33.md

```diff
@@ -0,0 +1,13 @@
+Known Issues
+============
+
+This page lists some known issues of the ArangoDB suite of products. 
+
+ArangoDB Customers can access the [_Technical & Security Alerts_](https://arangodb.atlassian.net/wiki/spaces/DEVSUP/pages/223903745) page after logging in the Support Portal.
```

Comment:
I think we would have to write "logging in in the Support Portal" or "logging into the Support Portal", but they both sound a bit weird. Therefore, I suggest to change it from verb to noun.
Also see https://www.englishforums.com/English/LogInIntoIntoOnto/vzdxc/post.htm
```suggestion
ArangoDB Customers can access the
[_Technical & Security Alerts_](https://arangodb.atlassian.net/wiki/spaces/DEVSUP/pages/223903745)
page after login in the Support Portal.
```